pkg/ingester/metrics.go

```diff
@@ -86,6 +87,10 @@ func newIngesterMetrics(r prometheus.Registerer, registerMetricsConflictingWithT
 			Name: "cortex_ingester_wal_replay_duration_seconds",
 			Help: "Time taken to replay the checkpoint and the WAL.",
 		}),
+		walCorruptionsTotal: prometheus.NewCounter(prometheus.CounterOpts{
+			Name: "cortex_ingester_wal_corruption_total",
```

Comment:
`cortex_ingester_wal_corruption_total` > `cortex_ingester_wal_corruptions_total` (added an `s` to `corruption`) to keep it consistent with TSDB